8.1/startup/generateStartupCommand.js

```diff
@@ -18,15 +18,45 @@ if (typeof process.env.WEBSITE_ROLE_INSTANCE_ID !== 'undefined'
     roleInstanceId = process.env.WEBSITE_ROLE_INSTANCE_ID;
 }
 
-var startupCommand = fs.readFileSync(CMDFILE, 'utf8').trim();
+// Is Application Insights enabled, with an associated ikey?
+var appInsightsEnabled = process.env.ENABLE_APPINSIGHTS && process.env.APPINSIGHTS_INSTRUMENTATIONKEY;
```

Comment:
Per our previous discussion, the expectation is that if the `ENABLE_APPINSIGHTS` is set, then an Application Insights resource has been provisioned, and its associated instrumentation key has been specified via the `APPINSIGHTS_INSTRUMENTATIONKEY` variable (which the Application Insights SDK auto-detects).